dataactbroker/fileRoutes.py

```diff
@@ -75,6 +75,17 @@ def finalize_submission(upload_id):
     def check_status(submission):
         return get_status(submission)
 
+    @app.route("/v1/gtas_window/", methods=["GET"])
+    def gtas_window():
+        gtas_window = get_gtas_window()
+
+        data = None
+
+        if gtas_window is not None:
```

Comment:
Would it work if you just did `if gtas_window`? This statement is equivalent to `if (variable != null)` which, as you know, is kind of redundant.